pkgs/by-name/ca/carburetor/package.nix

```diff
@@ -0,0 +1,76 @@
+{
+  lib,
+  fetchFromGitLab,
+  fetchpatch2,
+  python3Packages,
+  appstream,
+  desktop-file-utils,
+  gobject-introspection,
+  pkg-config,
+  meson,
+  wrapGAppsHook4,
+  libadwaita,
+  tractor,
+}:
+let
+  # This package should be updated together with pkgs/by-name/tr/tractor/package.nix
+  version = "5.0.0";
+in
+python3Packages.buildPythonApplication {
+
+  pname = "carburetor";
+  inherit version;
+
+  pyproject = false;
+
+  src = fetchFromGitLab {
+    domain = "framagit.org";
+    owner = "tractor";
+    repo = "carburetor";
+    rev = version;
+    hash = "sha256-Z67bqjogPz5sz6JwM68z1jsaqvRBAOMDeBLcyLo+QLY=";
+  };
+
+  patches = [
+    (fetchpatch2 {
+      url = "https://framagit.org/tractor/carburetor/-/commit/620b70288942497abc20ad26c043b593f66e9e3b.diff";
+      hash = "sha256-oFKLjvu+fwgyU4FIUb2K8jwXOP34P3pEazOhofwveJw=";
+    })
+  ];
+
```

Comment:
Why does it require this patch?

Reply:
It fixed the issue you mentioned (I also had it)

https://github.com/NixOS/nixpkgs/pull/309642#issuecomment-2155262313

Reply:
Oh wait no, there was another patch for tractor (you've deleted it?), it fixed that issue.

This patch fixes `no locale found` error

Reply:
https://github.com/flathub/io.frama.tractor.carburetor/issues/24

Reply:
Oh. sorry, I missed the tractor patch.